### DISCUSSION TITLE
How can I use distinct with a specific field of a column of type json?

### DISCUSSION THREAD

**Discussion by MML1357** [prisma/prisma#20300] [Q&A] [answered]
how can I use distinct with a specific field of a column of type json? For instance, the json column is named data and I have to use distinct with a field named code.

**Comment by ludralph** [comment, marked as answer]
Hi @MML1357 👋 

Prisma does not support using distinct directly on a specific field of a JSON column. You will need to make use of [Raw Queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases) in Prisma. You can leverage [SafeQL](https://safeql.dev/) to ensure you write the raw queries in a type-safe way.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
I came across this Stack Overflow question 😄 
https://stackoverflow.com/questions/76725781/with-prisma-how-to-perform-a-distinct-over-a-specific-field-of-a-json-column
